app/src/main/java/com/example/bisit/ui/customerMyReserve/CustomerMyReserveDetailFragment.kt

```
@@ -154,28 +160,34 @@ class CustomerMyReserveDetailFragment : Fragment() {
+                try {
+                    val api = RetrofitClient.getReservationApi(requireContext())
+                    val response = api.confirmReservation(reservationId)
+                    if (response.isSuccessful && response.body()?.success == true) {
+                        Toast.makeText(requireContext(), "시술이 확정되었습니다.", Toast.LENGTH_SHORT).show()
+                        // Refresh data
+                        fetchReservationDetail(reservationId)
+                        dialog.dismiss()
+                    } else {
+                        Toast.makeText(requireContext(), "시술 확정에 실패했습니다.", Toast.LENGTH_SHORT).show()
                     }
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)